ip-messaging/reachability-indicator/process-user-updates/meta.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Process a UserInfo Update",
-  "description": "Process a UserInfo Update and process the Reachability Indicators",
+  "title": "Process an UpdateReason",
+  "description": "Handle an UpdateReason change and process the Reachability Indicators",
```

Comment:
❓ I changed these strings because I found them while grep-ing for the file. What does a meta.json file do in the code sample process?

Reply:
I'm pretty sure that that they control the title displayed in the CMS in the header on the code sidebar. You can also see them when you pick an API Snippet to use in a page.